blogs/data-mover/data-mover-1.3/data-mover-1.3.0-overview.md

```diff
@@ -0,0 +1,230 @@
+
+For backup, Kopia's default uploader was extended to use the <a href=https://pkg.go.dev/github.com/kopia/kopia@v0.13.0/fs#StreamingFile>StreamingFile api</a> and mapping the block mode volume as device allows Kopia to correctly access the data and copy it to the Unified Repository. To restore, the block device data is copied back to the cluster via Kopia to a provisioned block device in /var/lib/kubelet/plugins by following a symbolic link to the device in /var/lib/kubelet/pods. <br>
+
+We would like to thank our partners <a href=https://cloudcasa.io/>CloudCasa</a> and  <a href=https://www.vmware.com/>VMware</a> for their collaboration and contributions in the upstream Velero project to enable this feature.
+Further improvements to block mode volumes are in progress to improve the utility and performance of the feature.  Please follow our work in the <a href=https://github.com/vmware-tanzu/velero>Velero Project</a> as we improve this critical feature.
+
+
+<h2>What Is CSI?</h2>
+
+<p dir="auto">One of the more important components of Data Mover to understand is CSI, or Container Storage Interface. CSI provides a layer of abstraction between container orchestration tools and storage systems such that storage vendors can develop a plugin once and have it work across a number of container orchestration systems. CSI defines an API for storage plugins to enable the creation of a snapshot that provides point-in-time snapshotting of volumes.</p>
+
+<p dir="auto">CSI compliant storage plugins are now the industry standard and are the preferred storage plugin type for most container orchestrators including Kubernetes. Most of Kubernetes "in-tree" drivers developed prior to CSI all have a target removal date as most storage vendors move towards deprecating non CSI plugins. However, issues concerning CSI volumes still remain. Some volumes have vendor-specific requirements, and can prevent proper portability and durability. Data Mover works to solve this case, which will be discussed more in the next section.</p>
+
+<p dir="auto">You can read more about CSI<span>&nbsp;</span><a href="https://kubernetes-csi.github.io/docs/">here</a>.</p>
+
+<h2>Components</h2>
+
+<h3><a href="https://github.com/openshift/oadp-operator">OADP OPERATOR</a>:</h3>
+
+<p dir="auto">OADP is the OpenShift API for Data Protection operator. This open source operator sets up and installs Velero on the OpenShift platform, allowing users to backup and restore applications. We will be installing Velero alongside the CSI plugin.</p>
```

Comment:
NIT: s/CSI plugin/Velero-plugin-for-csi

Reply:
so that users don't get confused with storage CSI plugin and velero CSI plugin

Reply:
We will be installing Velero alongside the velero-plugin-for-csi plugin.</p>

Reply:
thanks